index.js

```diff
@@ -383,7 +383,17 @@ class Raspi extends EventEmitter {
     }
   }
 
-  i2cConfig(delay) {
+  i2cConfig(options) {
+    var delay;
```

Comment:
No worries, we're all busy :) I went ahead and updated it on my end, so we're all good